cranelift/filetests/filetests/runtests/call.clif

```diff
@@ -66,3 +66,22 @@ block0(v0: b1):
 }
 ; run: %call_b1(true) == false
 ; run: %call_b1(false) == true
+
+
+
+; Tests calling across different calling conventions
+
+function %callee_fast_i64(i64) -> i64 fast {
```

Comment:
`fast` and `system_v` are actually the same in practice right now -- perhaps we could try `system_v` and `windows_fastcall` ?

Reply:
That might not work on `aarch64` and `s390x`, but we can try one of the `wasmtime` ones.